lib/matplotlib/dviread.py

```diff
@@ -989,7 +989,7 @@ def _parse_enc(path):
     with open(path, encoding="ascii") as file:
         no_comments = "\n".join(line.split("%")[0].rstrip() for line in file)
     array = re.search(r"(?s)\[(.*)\]", no_comments).group(1)
-    lines = [line for line in array.split("\n") if line]
+    lines = [line for line in array.split() if line]
```

Comment:
Why is this change?

Reply:
Because the previous version was buggy: on an encoding file like http://tug.ctan.org/info/fontname/8r.enc, you can see there are multiple entries per line (/.notdef /dotaccent /...) *and* lines are starting with a space, so the previous parser would fail immediately below when asserting that all lines start with a /, and even without the assertion, would misparse the file.